libraries/ms-common/test/test_chip_id.c

```diff
@@ -0,0 +1,32 @@
+#include "chip_id.h"
+
+#include <inttypes.h>
+#include "delay.h"
+#include "interrupt.h"
+#include "log.h"
+#include "misc.h"
+#include "soft_timer.h"
+#include "test_helpers.h"
+#include "unity.h"
+
+ChipId chip_id_1 = { 0 };
+ChipId chip_id_2 = { 0 };
```

Comment:
Could you change these to local variables? It's generally good practice to minimize global variables as much as possible.